tensorflow/python/lib/core/py_func.cc

```diff
@@ -500,6 +500,11 @@ class PyFuncOp : public OpKernel {
       call.ins.push_back(ctx->input(i));
     }
 
+    OP_REQUIRES(
+        ctx, Py_IsInitialized(),
+        errors::FailedPrecondition("Python interpreter state is not initialized. "
+                                   "The process may be terminated."));
+      
```

Comment:
Should we add a comment explaining why we have this check and that we are aware of (and seek suggestions) of the race between checking Py_IsInitialized() and assuming it is still true in the Py_* function calls below?

Reply:
Good idea! Done.